src/Playthrough.ts

```diff
@@ -66,6 +68,19 @@ class Playthrough extends EventEmitter<PlaythroughEvents> {
     this.baseUrl = baseUrl;
   }
 
+  public async getPlayerSessionId(): Promise<string> {
```

Comment:
this needs a safeguard against running forever, maybe limit to  100 attempts? And perhaps it should fail early if something like room doesn't exist